### DISCUSSION TITLE
Contribution chart is less accessible

### DISCUSSION THREAD

**Discussion by neel1996** [community/community#22204] [Accessibility]
<p>The contribution graph’s color scheme is not going along well with the dark mode which is making it hard to differentiate between each day’s contribution levels</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/1/11b696edc8e82e542b1a8d9f8ac3bdb393a29fc9.png" data-download-href="/uploads/short-url/2wHjqtgvXBg972e2p4QFdSqHQ6Z.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47709856/181069490-ba74fd9f-a9d3-446f-a0e7-e2a8ce90d529.png" alt="image" data-base62-sha1="2wHjqtgvXBg972e2p4QFdSqHQ6Z" width="690" height="150" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11b696edc8e82e542b1a8d9f8ac3bdb393a29fc9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">928×202 5.27 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by DisassembledSharper** [comment]
<p>In my opinion, I think purple would look better than green</p>

**Comment by gaknoll** [comment]
<p>Thanks <a class="mention" href="/neyuriki">@Neyuriki</a> for sharing the feedback! To make sure I’m sharing the right feedback with the team, would purple be a personal preference? Or is it due to another reason such as legibility or colorblindness etc… :slightly_smiling_face:</p>

**Comment by dgw** [comment]
<p>It could be personal preference. However, the third-party dark style I used for a long time changed the contributions graph cells to shades of blue.</p>
<p><em>Personally</em> I think that blue looks better on dark gray than green, but I freely admit that that’s a preference. Blue/gray combinations are definitely <em>not</em> recommended for text! :joy_cat:</p>

**Comment by ChibiKev** [comment]
<p>I also agree that the square colors in contributions should be changed for the dark mode.<br>
The green seems a bit too dark to see to notice any differences in commits.</p>

**Comment by gaknoll** [comment]
<p>Thanks <a class="mention" href="/dgw">@dgw</a> and <a class="mention" href="/chibikev">@ChibiKev</a> for adding your feedback to this topic as well. We’ve gone through many iterations of the contribution graph color scale over the past months but this feedback suggests that it’s worth us exploring it further :upside_down_face:</p>

**Comment by UchihaCFC** [comment]
<aside class="quote no-group" data-username="neel1996" data-post="1" data-topic="148250">
<div class="title">
<div class="quote-controls"></div>
 neel1996:</div>
<blockquote>
<p>The contribution graph’s color scheme is not going along well with the dark mode which is making it hard to differentiate between each day’s contribution levels</p>
</blockquote>
</aside>
<p>I think the same.</p>
<p>Maybe by making the background of not contribution days boxes somewhat lighter, like a gray similar to <span class="hashtag">#838383</span> ?</p>

**Comment by neel1996** [comment]
<p><a class="mention" href="/uchihacfc">@UchihaCFC</a> Exact thought. I also had a thought that the pumpkin color scheme which github rolled out for the Halloween could also be a good fit</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks <a class="mention" href="/neel1996">@neel1996</a> &amp; <a class="mention" href="/uchihacfc">@UchihaCFC</a> for the feedback on the contribution chart. We’ve been playing with the contrast on it quite a bit but have been hearing lots of feedback from the community that suggests we haven’t quite got it right yet! I’ll be sure to share this with the rest of the team</p>

**Comment by UchihaCFC** [comment]
<p><a class="mention" href="/neel1996">@neel1996</a> Yes, maybe the Halloween colors is a good option too, great idea.</p>

**Comment by UchihaCFC** [comment]
<p><a class="mention" href="/gaknoll">@gaknoll</a>  Thank you for considering our comments and working on improvements! :+1:</p>

**Comment by shreyanshsatvik** [comment]
<p>It’s difficult to notice the contribution graph with lesser amount of contribution of a particular day, i think a different color can be used or contrast can be modified .</p>

**Comment by gaknoll** [comment]
<p>Please don’t hesitate to share feedback and/or any bugs/issues you come across! We’re extremely grateful for our community in helping us get to dark mode GA :pray:</p>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/shreyanshsatvik">@shreyanshsatvik</a> - thanks for sharing this feedback! :pray:<br>
We’re hearing this from a lot of customers so we definitely plan on reviewing how we can improve the contribution graph. I’m going to merge this with a similar topic to keep the discussion in one place</p>

**Comment by shreyanshsatvik** [comment]
<p>Thanks <a class="mention" href="/gaknoll">@gaknoll</a> for hearing me out, I think tiger orange or tangerine orange or related colors will help</p>

**Comment by Amber-Pittman** [comment]
<p>Yes please! The only noticeable square I have is one from a few months ago, from the darkest shade. The mid-level shades of green blend in with the lightest shades that it almost appears as if I’ve barely done anything at all. :neutral_face:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/f/cfda61efae00ae3f88009e75fb28e2d7a69495a2.png" data-download-href="/uploads/short-url/tEKIzkdnCtJCZ08YDNZnllegpMu.png?dl=1" title="pleaseChangeShades" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/39929494/181069500-dcf7989b-39f4-4310-bf16-5c8ed7500c10.png" alt="pleaseChangeShades" data-base62-sha1="tEKIzkdnCtJCZ08YDNZnllegpMu" width="690" height="168" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cfda61efae00ae3f88009e75fb28e2d7a69495a2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">pleaseChangeShades</span><span class="informations">923×225 5.17 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by viviloaf** [comment]
<p>It becomes hard to see the default activity green squares against the dark background, and only the highest show up prominently. Maybe adding some type of border or shading, or contrasting the green again the dark with pastel colors would be better?</p>
<p>What does everyone else think?</p>

**Comment by gaknoll** [comment]
<p>Hi <a class="mention" href="/vnguye34">@vnguye34</a> - thanks for sharing your feedback on the contribution graph! We’ve had quite a bit of feedback come in from the community on this (I’ll merge this with the other topic in a minute) and we’ve already started work on a new iterations of a design to improve the contrast</p>

**Comment by cehfisher** [comment]
Hello @neel1996 - I wanted to follow up on this issue and see if the recent dark mode theme changes have been helpful based on your initial feedback. If they haven't addressed the problem, could you please let me know what additional changes are needed? (cc @queenofcorgis)
